cmd/kubectl-gadget/utils/flags.go

```diff
@@ -114,6 +117,34 @@ func AddCommonFlags(command *cobra.Command, params *CommonFlags) {
 			}
 		}
 
+		// Verify if the node specified in the filter actually exist. This check
+		// will be removed when we will support the addition/deletion of nodes.
+		if params.Node != "" {
+			client, err := k8sutil.NewClientsetFromConfigFlags(KubernetesConfigFlags)
+			if err != nil {
+				return fmt.Errorf("impossible to verify node parameter: "+
+					"error setting up a Kubernetes client: %q", err)
+			}
+
+			nodes, err := client.CoreV1().Nodes().List(context.TODO(), metaV1.ListOptions{})
+			if err != nil {
+				return fmt.Errorf("impossible to verify node parameter: "+
+					"error retrieving cluster nodes: %q", err)
```

Comment:
Breaking lines is good but not error messages. Something like `git grep "impossible to verify node parameter: error retrieving cluster node` won't provide any results, and it's very annoying if you want to know the source of the error.

Also I think this could be shortened to:

```suggestion
				return fmt.Errorf("failed to check if node exists: %v", err)
```

Reply:
I usually look for errors per sentence separated by `:`. For instance, I look for `impossible to verify node parameter` and then for `error retrieving cluster node` because if they are usually composed of different return functions. However, I understand what you mean so I will update the PR.